src/container.php

```diff
@@ -161,6 +161,7 @@
   ->addMethodCall('addSubscriber', array(new Reference('app.listener.channel')))
   ->addMethodCall('addSubscriber', array(new Reference('console.listener.milestone')))
   ->addMethodCall('addSubscriber', array(new Reference('console.listener.favourite')))
+  ->addMethodCall('addSubscriber', array(new Reference('console.listener.exception')))
```

Comment:
It looks like is not enough to register the console.listener.exception, but that we must add it to the `addSubscriber` method of the `Symfony\Component\EventDispatcher\EventDispatcher`